cloudigrade/util/aws/helper.py

```diff
@@ -122,6 +131,12 @@ def _verify_policy_action(session, action):
                 Attribute='createVolumePermission',
                 OperationType='add',
             )
+        elif action in cloudtrail_actions:
```

Comment:
This is just a _stylistic_ suggestion that lets us drop the `cloudtrail_actions` list above. If you don't like it, keep what you have. 🙂  Instead of checking that the action is in the predefined `cloudtrail_actions` list, since we are relatively confident that _all_ cloudtrail actions cannot be verified, maybe change this to the following?

```
elif action.startswith('cloudtrail:'):
```

_If_ you do that, then I'd move the block comment from lines 104-112 to inside this conditional so it lives with its relevant logic.

Reply:
Great suggestion 😄